### DISCUSSION TITLE
Does the vmc exploit still work?

### DISCUSSION THREAD

**Discussion by OPcoolman** [3kh0/ext-remover#1168] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
So I tried the vmc exploit and it didn't work for me on version 128, is it patched?

**Comment by HelloIloveGimkit2** [comment]
yes it does but cr0sh may be blocked here's a way to unblock it: https://github.com/3kh0/ext-remover/discussions/1154

**Reply by HelloIloveGimkit2** [reply]
btw with the cr0sh unblocker you gotta be fast and open it as fast as you can

**Reply by OPcoolman** [reply]
Are you sure when I type vmc I see a bunch of errors

**Reply by HelloIloveGimkit2** [reply]
oh then it's blocked for you

**Reply by HelloIloveGimkit2** [reply]
for me it still works

**Reply by OPcoolman** [reply]
It says command failed

**Reply by OPcoolman** [reply]
Show me what you see

**Reply by HelloIloveGimkit2** [reply]
can't cuz I don't wanna reset my Chromebook to reload the crosh policy

**Reply by HelloIloveGimkit2** [reply]
it's not blocked for me reason: your school prob disabled the vmc command on crosh or vmc isn't supported on your chromebook

**Reply by OPcoolman** [reply, marked as answer]
it got patched on version 123
